src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```
@@ -49,6 +50,11 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         try {
             $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, false);
         } catch (\Exception $e) {
+        } catch (\Throwable $e) {
+            $e = new FatalThrowableError($e);
```

should be removed (let it bubble yes)